src/js/components/Button/Button.js

```diff
@@ -169,6 +171,20 @@ const Button = forwardRef(
       );
     }
 
+    const sendAnalytics = useAnalytics();
+
+    const onClick = useCallback(
+      (event) => {
+        sendAnalytics({
+          type: 'buttonClick',
+          element: event.target,
```

Comment:
This comes through as the actual target (e.g. the child svg, text, div, etc.), but I find myself wanting to know what the parent button is.